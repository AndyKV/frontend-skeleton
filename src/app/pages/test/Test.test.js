import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import Test from './Test'

describe('component Test', function() {
  it('renders correctly', function() {
    const component = renderer.create(<Test />).toJSON()

    expect(component).toMatchSnapshot()
  })

  it('should not contain h2 headline with text', function() {
    const wrapper = shallow(<Test />)
    const text = <h2>Text to test</h2>

    expect(wrapper.contains(text)).toEqual(false)
  })
})
