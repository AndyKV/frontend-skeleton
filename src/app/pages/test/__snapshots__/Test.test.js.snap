// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component Test renders correctly 1`] = `<div />`;
